chaste_codegen/templates/Cvode/cpp/ComputeDerivedQuantities

```diff
@@ -5,14 +5,19 @@
         // Inputs:
         // Time units: millisecond
         {% for state_var in state_vars %}
-        {%- if state_var.in_derived_quant %}double {{ state_var.var }} = NV_Ith_S(rY,{{loop.index0}});
+        {%- if state_var.in_derived_quant %}double {{ state_var.var }} = {% if loop.index0 == membrane_voltage_index %}(mSetVoltageDerivativeToZero ? this->mFixedVoltage : NV_Ith_S(rY, {{loop.index0}}));{%- else %}NV_Ith_S(rY, {{loop.index0}});{%- endif %}
         // Units: {{state_var.units}}; Initial value: {{state_var.initial_value}}
         {% endif %}{%- endfor %}
-
         // Mathematics
-        {%- for eq in derived_quantity_equations %}
-        const double {{eq.lhs}} = {{eq.rhs}}; // {{eq.units}}
-        {%- endfor %}
+        {%- for eq in derived_quantity_equations %}{%- if eq.is_data_clamp_current is defined and eq.is_data_clamp_current %}
+        // Special handling of data clamp current here (see #2708)
```

Comment:
Not sure if it's sensible to include Chaste Trac issue references in a GitHub repo?

Reply:
huh? oo copied-and-pasted too much